Druid-Tests/DRCompilationUnitTest.class.st

```diff
@@ -0,0 +1,143 @@
+Class {
+	#name : #DRCompilationUnitTest,
+	#superclass : #TestCase,
+	#category : #'Druid-Tests-CompilerBuilder'
+}
+
+{ #category : #tests }
+DRCompilationUnitTest >> compilerClass [
+
+	^ Smalltalk globals at: self compilerClassName
+]
+
+{ #category : #tests }
+DRCompilationUnitTest >> compilerClassName [
+
+	^ #MyJITCompilerClass
+]
+
+{ #category : #'instance creation' }
+DRCompilationUnitTest >> newEmptyCompilationUnit [
+
+	self flag: #TODO. "Duplicated from compilationUnitForTesting"
+
+	^ DRInterpreterCompilationUnit new
+		compilerBuilder: (DRInterpreterToCompiler fromInterpreterClass: DruidTestInterpreter);
+		targetClass: self compilerClassName;
+		yourself
+]
```

Comment:
Inherits from a better TestCase to avoid all this configuration

Reply:
Not such a big deal I think, it's a helper, right?